js/views/userShort.js

```diff
@@ -0,0 +1,109 @@
+import $ from 'jquery';
+import BaseVw from './baseVw';
+import loadTemplate from '../utils/loadTemplate';
+import app from '../app';
+import followUtils from '../utils/follow';
+import Profile from '../models/Profile';
+
+export default class extends BaseVw {
+  constructor(options = {}) {
+    super(options);
+    this.options = options;
+    this.guid = options.guid || this.model.get('guid');
+    this.ownGuid = this.guid === app.profile.id;
+    this.profileArgs = {}; // create blank for placeholder render
+    this.followedByYou = app.ownFollowing.get(this.guid) !== undefined;
```

Comment:
You are using a followUtil method for this determination in other places. For consistency, should we use it here?